keylime/tpm/tpm_main.py

```diff
@@ -505,6 +505,12 @@ def __take_ownership(self, config_pw):
             logger.info("Taking ownership with config provided TPM owner password")
             owner_pw = config_pw
 
+        logger.debug("Removing all saved sessions from TPM")
+        if self.tools_version == "3.2":
+            retDict = self.__run(["tpm2_flushcontext", "-s"], raiseOnError=False)
+        elif self.tools_version in ["4.0", "4.2"]:
```

Comment:
Is there a reason to have an if/elif rather than to have the three versions in a single list, considering both cases result in exactly the same action from what i can tell?

Reply:
you are right, there is not...I was under the impression that `tpm2_flushcontext` had a slight variation in parameter between 3.2 and 4.X, but that was not the case.

Reply:
lgtm after removing the conditional

Reply:
agreed